sfs/mono/source.py

```diff
@@ -395,7 +397,9 @@ def point_image_sources(omega, x0, grid, L, max_order, coeffs=None, c=None):
     p = 0
     for position, strength in zip(xs, source_strengths):
         if strength != 0:
-            p += strength * point(omega, position, grid, c)
+            # `point` can return NaN, which is fine
```

Comment:
Is it really NaN?
Isn't it "complex infinity"?

Reply:
Changed it to
```python
# point can be complex infinity
```